### ISSUE TITLE
[4.x]: Update Order Status element action missing, for mutliple users

### ISSUE THREAD

**Issue by bossanova808** [craftcms/commerce#3362]
### Description

My own user, super admin, has this order element index option:

![image](https://github.com/craftcms/commerce/assets/731309/6fc3e7e7-a210-471f-b788-b6e85c803c73)

Other admin users (staff) do NOT see this option, but they do have what I believe are all the needed Commerce permissions:

![image](https://github.com/craftcms/commerce/assets/731309/fb0a42c8-e319-4ee0-a77d-488424257fb9)

I can't find anything else about this except for an ancient SE report:
https://craftcms.stackexchange.com/questions/25875/update-order-status-action-missing

And the docs don't really indicate anything extra is required:
https://craftcms.com/docs/commerce/4.x/custom-order-statuses.html#in-the-control-panel

It seems odd to me those two permissions 'Edit Orders' and 'Delete Orders' are visually different to all the rest - it's not clear to me why the rest are greyed out really either.

### Craft CMS version

4.6

### Craft Commerce version

4.3.3

### PHP version

8.2.14

### Operating system and version

Ubuntu 22.04 (Forge)

### Database type and version

MySQL 8.0.35

### Image driver and version

Imagick 3.7.0 (ImageMagick 6.9.11-60)

### Installed plugins and versions

![image](https://github.com/craftcms/commerce/assets/731309/acb4f3f8-0fa3-4662-82ff-5a1323f3e3bb)

**Comment by nfourtythree** [comment]
Hi @bossanova808 

Thank you for your report.

I have created a user group with the permissions as you mentioned:

<img width="343" alt="CleanShot 2024-03-26 at 08 54 11@2x" src="https://github.com/craftcms/commerce/assets/266453/90f9f062-81db-4a3e-b9b3-40bf752c3269">

I then logged in as that person, went to the order index and was able to see and use the update order status action.

<img width="868" alt="CleanShot 2024-03-26 at 08 54 43@2x" src="https://github.com/craftcms/commerce/assets/266453/ebcc1805-dd5e-4e49-bb82-cd07614b7970">

Have you tried:

- disabling all the other plugins? 
- removing any custom code? 
- if you are able to replicate the issue on a fresh install of Craft/Commerce?

Thanks!
